src/rockstor/storageadmin/static/storageadmin/js/views/share_details_layout_view.js

```diff
@@ -94,6 +94,10 @@ ShareDetailsLayoutView = RockstorLayoutView.extend({
 		var current_appliance = this.appliances.find(function(appliance) {
 			return appliance.get('current_appliance') == true;
 		});
+        var systemShare = false;
+        if (parseInt(this.share.get('id')) < 3 && this.share.get('pool')['id'] == '1') {
```

Comment:
Can we tighten this logic @MFlyer ? It requires backend changes. There's a role attribute in the Pool model which can be updated as part of the Pool state refresh. We should rely on that attribute rather than the Pool's id being 1. Same sort of business logic should apply in identifying /home and /root Shares. In fact, the OS Pool may have a different name, but home and root almost never do. So you can just check the names. But to make it 100% solid, we can have a property in the Share model that does a runtime check if the subvolume is mounted as /home or /root.